src/inmanta/module.py

```diff
@@ -556,6 +596,53 @@ def install(self, project: "Project", module_spec: List[InmantaModuleRequirement
             raise InvalidModuleException(f"{python_package} does not contain a {namespace_package} module.")
         return self.from_path(project, module_name, path)
 
+    def log_pre_install_information(self, project: "Project", module_name: str) -> None:
+        LOGGER.debug("Installing module %s (v2).", module_name)
+
+    @staticmethod
+    def take_v2_modules_snapshot() -> Dict[str, "Version"]:
+        """
+        Log and return a dictionary containing currently installed v2 modules and their versions
+        """
+        version_snapshot = env.PythonWorkingSet.get_packages_in_working_set(inmanta_modules_only=True)
+        if version_snapshot:
+            out = ["Snapshot of modules versions pre-install:"]
```

Comment:
This message is specific to the pre-installation check. Either it shouldn't be, or it should mention this as a usage constraint in the docstring.